### ISSUE TITLE
YAML syntax error in "Pharos Popup Client.jss.recipe.yaml"

### ISSUE THREAD

**Issue by grahampugh** [autopkg/MLBZ521-recipes#42]
I guess you have some tabs instead of spaces in your yaml...

```
WARNING: yaml error for /Users/serveradmin/Library/AutoPkg/RecipeRepos/com.github.autopkg.MLBZ521-recipes/Pharos Popup Client/Pharos Popup Client.jss.recipe.yaml: while scanning for the next token
found character '\t' that cannot start any token
```

**Comment by grahampugh** [comment]
Incidentally, I'm curious as to why you are spending time rewriting JSS recipes to YAML? Most likely JSS recipes won't work anymore after late 2022.

**Comment by grahampugh** [comment]
I recommend the YAML extension in Visual Studio Code. This automatically lints any YAML when you save the file.

**Comment by MLBZ521** [comment]
Yeah, I rushed the conversion on these and failed to validate the formatting, but all the (breaking) formatting issues should be resolved as of e3baad3.  I'll still accept your PR for the additional items.

Currently, I'm still using JSSImporter in my environment.  Our AutoPkg process heavily relies on JSSImporter.  From our standard recipes, to software that isn't publicly downloadable and use my [OfflineApps.py](https://github.com/autopkg/MLBZ521-recipes/blob/master/Shared%20Processors/OfflineApps.py) processor, and then I have a large Slack integration that currently is built on the information that JSSImporter feeds to and from it.

I'm planning to convert away from JSSImporter, given I have the time to do so eventually.  If I have too, I'll make JSSImporter work with OAuth.  But that will not be my primary plan, seeing as the community is moving away from it.  I have a bit of other code that will need to be updated to support OAuth as well....along with a lot of other projects that I'm working on.

And yes, I've been meaning to add a YAML extension if VS Code, I'll do that today -- thanks.

**Comment by MLBZ521** [comment]
I think I left this open to remind me to add a YAML extension to VS Code....which I did a long time ago...  So going to close this now.
